packages/amplication-server/src/core/entity/entity.resolver.ts

```diff
@@ -120,9 +120,6 @@ export class EntityResolver {
     @Parent() entity: Entity,
     @Args() args: FindManyEntityFieldArgs
   ) {
-    if (entity.fields && entity.fields.length) {
```

Comment:
I believe it's not needed anymore. 
The last commit that modifies these lines is 330716fcf993ae03f62de474a749d0f81f201db5 and at that time the `entity` resolver could return fields : 

```ts
// from entity.service.ts
async entity(args: FindOneEntityArgs): Promise<Entity | null> {
    let version, findArgs;
    ({ version, ...findArgs } = args);
    // return this.prisma.entity.findOne(findArgs);

    //
    let entityVersion = await this.getEntityVersion(
      args.where.id,
      args.version
    );

    if (!entityVersion) {
      throw new NotFoundException(`Cannot find entity`); //todo: change phrasing
    }

    let entity: Entity = await this.prisma.entity.findOne(findArgs);

    entity.versionNumber = entityVersion.versionNumber;

    entity.fields = await this.getEntityFields(entity);

    return entity;
  }
``` 
but now it can't return fields anymore, since there is no `include` in the prisma query:
```ts
  async entity(args: FindOneEntityArgs): Promise<Entity | null> {
    const entity = await this.prisma.entity.findFirst({
      where: {
        id: args.where.id,
        deletedAt: null
      }
    });

    if (!entity) {
      throw new AmplicationError(
        `Cannot find entity where ${JSON.stringify(args.where)}`
      );
    }

    return entity;
  }
```

Reply:
He is right, the entity that return from the the entity in entity.service.ts isn't able to return fields anymore.